docs/install.md

```diff
@@ -25,9 +25,26 @@ conda install pytorch=1.5.0 torchvision cudatoolkit=10.1 -c pytorch
 ```
 
 c. Install [MMCV](https://mmcv.readthedocs.io/en/latest/) following the [official instructions](https://mmcv.readthedocs.io/en/latest/#installation).
-Either `mmcv` or `mmcv-full` is compatible with MMSegmentation, but for methods like CCNet and PSANet, CUDA ops in `mmcv-full` is required
+Either `mmcv` or `mmcv-full` is compatible with MMSegmentation, but for methods like CCNet and PSANet, CUDA ops in `mmcv-full` is required.
```

Comment:
We may separate Windows and Linux. 
And put Linux ahead.

Reply:
OK.